game/index.html

```diff
@@ -19,7 +19,7 @@
     </div>
 
     <div class="game-board">
-
+        <div id=ludek></div>
```

Comment:
nazywajmy klasy, id i inne rzeczy spójnie tzn. jeśli przyjęliśmy konwencję po angielsku to się tego trzymajmy.